policy/modules/kernel/filesystem.if

```diff
@@ -5046,6 +5046,25 @@ interface(`fs_relabel_tmpfs_blk_file',`
 	relabel_blk_files_pattern($1, tmpfs_t, tmpfs_t)
 ')
 
+########################################
+## <summary>
+##	Relabel named pipes on tmpfs filesystems.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`fs_relabel_tmpfs_fifo_file',`
```

Comment:
There is some inconsistency in the existing interfaces. It should be `fs_relabel_tmpfs_fifo_files` (plural)

Reply:
I can see the same inconsistency in the sibling interfaces (`fs_relabel_tmpfs_chr_file` & `fs_relabel_tmpfs_blk_file`). Would you like me to throw in a commit to align those as well?

Reply:
BTW, it would be worth to mention (for the future) on the [InterfaceNaming](https://github.com/SELinuxProject/refpolicy/wiki/InterfaceNaming) wiki page that predicates should be used in plural form.

Reply:
Thanks for note on the interface naming page. Fixing the inconsistency isn't necessary for the PR, but if you'd like to you fix that, it would be great. You'd need to have deprecated versions of the old interfaces (grep deprecated in the policy for examples).